taxonomy-manager-rest-server/src/main/java/com/digirati/taxman/rest/server/taxonomy/storage/ConceptDao.java

```diff
@@ -29,6 +29,11 @@ public ConceptDao(DataSource dataSource) {
         return jdbcTemplate.query("SELECT * FROM get_all_concepts()", recordMapper).stream();
     }
 
+    public Stream<ConceptRecord> getConceptsByPartialLabel(String label) {
```

Comment:
If we already have a full collection, we shouldn't return a `Stream`. Let the API consumer decide if they want a stream or a collection.

If they want to go back to a collection from a `Stream`, they have to copy to a new collection. Going to a `Stream` from a collection is zero-copy.

Reply:
This was to be consistent with the method above (`'loadAllRecords`). I agree with your point in principle but in this case there's a single consumer of this which does want this as a Stream